intel/isolate.py

```diff
@@ -21,9 +21,21 @@
 import subprocess
 
 ENV_CPUS_ASSIGNED = "CMK_CPUS_ASSIGNED"
+ENV_CPUS_ASSIGNED_MASK = "CMK_CPUS_ASSIGNED_MASK"
 ENV_CPUS_INFRA = "CMK_CPUS_INFRA"
 ENV_NUM_CORES = "CMK_NUM_CORES"
 
+def convert_array2bitmask(array):
```

Comment:
I think it would be better if this function was moved to the config.go file as part of the CPUList class,

The function and implementation itself looks good though,

Reply:
Do you mean config.py? 
CPUList class, as_bitmask(self) method? Or as_bitmask(self, array)?
May be reasonable to add as_list(self) function too? Or as_array(self) - it cat initialize CPUList.array field in first invocation and return it, and on next invocations return CPUList.array. Is it reasonable?

Reply:
@lmdaly and @LilyaBu note that there are multiple CPUList objects that are to be included in the env variables, so returning bitmask from a single CPUList object and then merging it with other bitmasks returned from other CPULists may be a bit too complex

personally, I'd suggest to simply move this function to the utils package

Reply:
including the utils package won't introduce any dependencies for the binary I assume?